onnx/external_data_helper.py

```diff
@@ -258,8 +256,9 @@ def write_external_data_tensors(model, filepath):  # type: (ModelProto, Text) ->
     The modified model object.
     """
     for tensor in _get_all_tensors(model):
-        if uses_external_data(tensor):
+        if uses_external_data(tensor) and tensor.HasField("raw_data"):
```

Comment:
add some comments explaining why you are checking for uses_external_data. It is not clear

Reply:
more elaborate would be nice - something like
Writing to external data happens in 2 passes. In first pass tensors with raw data which pass the necessary conditions (size threshold etc) are marked for serialization. In the second pass, the raw data in these tensors is serialized to a file. This is the second step. Serialize if tensor has raw data and it was marked for serialization.

Reply:
Updated. Thanks